lib/jekyll/commands/build.rb

```diff
@@ -57,7 +57,7 @@ def build(site, options)
           Jekyll.logger.info "Incremental build:", (full_build ? "disabled" : "enabled")
           Jekyll.logger.info "Generating..."
           process_site(site)
-          Jekyll.logger.info "", "done in #{Time.now - t} seconds."
+          Jekyll.logger.info "", "done in #{(Time.now - t).round 2} seconds."
```

Comment:
Is there a possible world which shows `0.00` seconds? Maybe 3?

Also, from a code-style standpoint, I'd prefer parentheses around the argument to `#round` :smile: